lib/internal/test_runner/coverage.js

```diff
@@ -136,7 +133,7 @@ class TestCoverage {
     const coverage = this.getCoverageFromDirectory();
     const coverageSummary = {
       __proto__: null,
-      workingDirectory: this.workingDirectory,
+      workingDirectory: this.globalOptions.cwd,
```

Comment:
It is exhausting to keep asking this, but please keep changes to a minimum. I agree that this refactoring should happen, but it is not related to the commit's goal of "require --enable-source-maps for sourcemap coverage"

Reply:
Sorry! I don't mean to make you keep asking this of me, I'll heed this in the future, and modify this PR.